src/components/accordion/__tests__/__snapshots__/CdrAccordion.spec.js.snap

```diff
@@ -1,6 +1,6 @@
 // Vitest Snapshot v1
 
-exports[`CdrAccordion > renders correctly 1`] = `
+exports[`CdrAccordion > when mounted > renders correctly 1`] = `
```

Comment:
When the test structure changes but the markup remains the same, it becomes "obsolete". Interestingly the snapshots still work as expected